video_iframe/public/js/src/video_iframe.js

```diff
@@ -0,0 +1,56 @@
+/* Javascript for VideoIframeXBlock. */
+function VideoIframeXBlock(runtime, element, data) {
+
+    function updateCount(result) {
+        $('.count', element).text(result.count);
+    }
+
+    var handlerUrl = runtime.handlerUrl(element, 'increment_count');
+
+    $('p', element).click(function(eventObject) {
+        $.ajax({
+            type: "POST",
+            url: handlerUrl,
+            data: JSON.stringify({"hello": "world"}),
+            success: updateCount
+        });
+    });
+
+    wrapper_width = $('.video_wrapper', element).width();
+    $('.video_wrapper', element).height(wrapper_width*0.6);
+
+    video_download_link = data['video_download_link'];
+    captions_download_link = data['captions_download_link'];
+    display_name = data['display_name']
+    iframe_link = data['iframe_link']
+
+    if(video_download_link || video_download_link!==""){
```

Comment:
Can you please explain why do we need ` video_download_link!==""`?

Reply:
For abundance of caution I guess, no other reason to justify this. Removed. :P